### PR TITLE
Fix blink issues when highlight file items

### PR DESCRIPTION
**Pull request by guanglinn**
Improved the code related to highlight file items. See [PR 2213](https://github.com/gsantner/markor/pull/2213#issuecomment-2026327489).

### CODE REVIEW

File: app/src/main/java/net/gsantner/opoc/util/GsContextUtils.java [hunk 1]
```diff
@@ -2880,8 +2881,19 @@ public boolean isDarkModeEnabled(final Context context) {
...
     public static void blinkView(final View view) {
         if (view != null) {
-            final float init = view.getAlpha();
-            ObjectAnimator.ofFloat(view, View.ALPHA, init, 0.1f, 1.0f, 0.1f, 1.0f, init).setDuration(1000).start();
+            final Object tag = view.getTag(R.id.view_animator);
+            if (tag != null && tag instanceof ObjectAnimator) {
+                if (((ObjectAnimator) tag).isRunning()) {
+                    return;
+                } else {
+                    view.setTag(R.id.view_animator, null);
```
Comment (harshad1) [inline app/src/main/java/net/gsantner/opoc/util/GsContextUtils.java:2889]: I believe this function is only used to blink the view in showFile. 

I am 100% ok with replacing this use with your approach to highlighting the background. 

Either way this is a good change.

File: app/src/main/java/net/gsantner/opoc/util/GsContextUtils.java [hunk 2]
```diff
@@ -128,6 +128,7 @@
 
 import com.google.android.material.snackbar.Snackbar;
```
Comment (gsantner) [inline app/src/main/java/net/gsantner/opoc/util/GsContextUtils.java:130]: Dont import app specific ID here (markor R). If possible make the added thing in code dynamically, or lookup the id by the resources helpers that are here.

**Comment by guanglinn** [comment]
I made some improvements, I think it is OK now.

**Comment by guanglinn** [comment]
https://github.com/gsantner/markor/assets/60415467/b2fc2d4e-5965-4c96-a2a2-dba790024652

----

https://github.com/gsantner/markor/assets/60415467/229a750d-3b42-47c6-84c7-d4b9a582f938

**Comment by gsantner** [comment]
Thanks but basically superseed by #2262 now right?

**Comment by harshad1** [comment]
> Thanks but basically superseed by #2262 now right?

I believe so, yes

**Comment by harshad1** [comment]
Closing now. @guanglinn We can resurrect this or reuse this code if it becomes necessary. Thanks :)

**Comment by guanglinn** [comment]
@harshad1 OK. I'm sorry for not reviewing your code in time due to some personal matters.
Yes, I can resurrect some code if there is still room for improvement.

**Comment by gsantner** [comment]
As of blinking highlight at least, the solution on master branch now worked very fine

**Pull request closed without merging**